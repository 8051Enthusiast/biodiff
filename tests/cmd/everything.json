{
  "version": "1",
  "presets": {
    "global": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": {
          "Blockwise": 8192
        },
        "backend": {
          "backend": "wfa2"
        }
      },
      {
        "name": "global",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Global",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_global": 0,
    "semiglobal": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Semiglobal",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_semiglobal": 0
  },
  "style": {
    "mode": "Decimal",
    "ascii_col": true,
    "bars_col": true,
    "layout": "Horizontal",
    "spacer": true,
    "right_to_left": true,
    "column_count": {
      "Fixed": 13
    },
    "no_scroll": false
  }
}