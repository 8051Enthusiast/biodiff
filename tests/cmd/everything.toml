bin.name = "biodiff"
args = ["--color", "--config", "tests/cmd/everything.json", "--print", "-gglobal", "tests/text_x.txt", "tests/text_y.txt"]
status.code = 0