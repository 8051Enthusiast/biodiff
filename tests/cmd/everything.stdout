[38;5;8m [39m 32 101 109 111 115  32 115 105   32 115 105 104  84 [38;5;8m0000  |[39m emos si sihT[38;5;8m |[39m▁▄▄▄▄▁▄▄▁▄▄▄▃[38;5;8m | [39m 32 101 109 111 115  32 115 105   32 115 105 104  84 [38;5;8m0000  |[39m emos si sihT[38;5;8m |[39m▁▄▄▄▄▁▄▄▁▄▄▄▃
[38;5;8m [39m 98  32 116 120 101 116  32 110  111 109 109 111  99 [38;5;8m000d  |[39mb txet nommoc[38;5;8m |[39m▄▁▄▄▄▄▁▄▄▄▄▄▄[38;5;8m | [39m 98  32 116 120 101 116  32 110  111 109 109 111  99 [38;5;8m000d  |[39mb txet nommoc[38;5;8m |[39m▄▁▄▄▄▄▁▄▄▄▄▄▄
[38;5;8m [39m102  32 104 116 111  98  32 110  101 101 119 116 101 [38;5;8m001a  |[39mf htob neewte[38;5;8m |[39m▄▁▄▄▄▄▁▄▄▄▄▄▄[38;5;8m | [39m102  32 104 116 111  98  32 110  101 101 119 116 101 [38;5;8m001a  |[39mf htob neewte[38;5;8m |[39m▄▁▄▄▄▄▁▄▄▄▄▄▄
[38;5;8m [38;5;10m101 116  32 115 105 104  84  32  [39m 46 115 101 108 105 [38;5;8m0027  |[38;5;10met sihT [39m.seli[38;5;8m |[38;5;10m▄▄▁▄▄▄▃▁[39m▂▄▄▄▄[38;5;8m | [38;5;10m                                 [39m 46 115 101 108 105 [38;5;8m0027  |[38;5;10m        [39m.seli[38;5;8m |[38;5;10m░░░░░░░░[39m▂▄▄▄▄
[38;5;8m [38;5;10m110 105  32 121 108 110 111  32  115 105  32 116 120 [38;5;8m0034  |[38;5;10mni ylno si tx[38;5;8m |[38;5;10m▄▄▁▄▄▄▄▁▄▄▁▄▄[38;5;8m | [38;5;10m                                                     [38;5;8m002c  |[38;5;10m             [38;5;8m |[38;5;10m░░░░░░░░░░░░░[39m
[38;5;8m [38;5;10m102  32 116 115 114 105 102  32  115 105 104 116  32 [38;5;8m0041  |[38;5;10mf tsrif siht [38;5;8m |[38;5;10m▄▁▄▄▄▄▄▁▄▄▄▄▁[38;5;8m | [38;5;10m                                                     [38;5;8m002c  |[38;5;10m             [38;5;8m |[38;5;10m░░░░░░░░░░░░░[39m
[38;5;8m [39m114  97 104  99  32 101 104  84   10 [38;5;10m 46 101 108 105 [38;5;8m004e  |[39mrahc ehT.[38;5;10m.eli[38;5;8m |[39m▄▄▄▄▁▄▄▃▁[38;5;10m▂▄▄▄[38;5;8m | [39m114  97 104  99  32 101 104  84   10 [38;5;10m                [38;5;8m002c  |[39mrahc ehT.[38;5;10m    [38;5;8m |[39m▄▄▄▄▁▄▄▃▁[38;5;10m░░░░[39m
[38;5;8m [39m110 105  32 [38;5;9m 88 [39m 32 115 105  32  114 101 116  99  97 [38;5;8m005b  |[39mni [38;5;9mX[39m si retca[38;5;8m |[39m▄▄▁[38;5;9m▃[39m▁▄▄▁▄▄▄▄▄[38;5;8m | [39m110 105  32 [38;5;9m 89 [39m 32 115 105  32  114 101 116  99  97 [38;5;8m0035  |[39mni [38;5;9mY[39m si retca[38;5;8m |[39m▄▄▁[38;5;9m▃[39m▁▄▄▁▄▄▄▄▄
[38;5;8m [39m 83  10  46 [38;5;9m 89 [39m 32 102 111  32  100  97 101 116 115 [38;5;8m0068  |[39mS..[38;5;9mY[39m fo daets[38;5;8m |[39m▃▁▂[38;5;9m▃[39m▁▄▄▁▄▄▄▄▄[38;5;8m | [39m 83  10  46 [38;5;9m 88 [39m 32 102 111  32  100  97 101 116 115 [38;5;8m0042  |[39mS..[38;5;9mX[39m fo daets[38;5;8m |[39m▃▁▂[38;5;9m▃[39m▁▄▄▁▄▄▄▄▄
[38;5;8m [39m116 120 101 116  32 101 114 111  109  32 101 109 111 [38;5;8m0075  |[39mtxet erom emo[38;5;8m |[39m▄▄▄▄▁▄▄▄▄▁▄▄▄[38;5;8m | [39m116 120 101 116  32 101 114 111  109  32 101 109 111 [38;5;8m004f  |[39mtxet erom emo[38;5;8m |[39m▄▄▄▄▁▄▄▄▄▁▄▄▄
[38;5;8m [39m101 114 117 115  32 101 107  97  109  32 111 116  32 [38;5;8m0082  |[39merus ekam ot [38;5;8m |[39m▄▄▄▄▁▄▄▄▄▁▄▄▁[38;5;8m | [39m101 114 117 115  32 101 107  97  109  32 111 116  32 [38;5;8m005c  |[39merus ekam ot [38;5;8m |[39m▄▄▄▄▁▄▄▄▄▁▄▄▁
[38;5;8m [39m111  32 114 101  98 109 117 110   32 101 104 116  32 [38;5;8m008f  |[39mo rebmun eht [38;5;8m |[39m▄▁▄▄▄▄▄▄▁▄▄▄▁[38;5;8m | [39m111  32 114 101  98 109 117 110   32 101 104 116  32 [38;5;8m0069  |[39mo rebmun eht [38;5;8m |[39m▄▁▄▄▄▄▄▄▁▄▄▄▁
[38;5;8m [39m 99  97 114  97 104  99  32 101  109  97 115  32 102 [38;5;8m009c  |[39mcarahc emas f[38;5;8m |[39m▄▄▄▄▄▄▁▄▄▄▄▁▄[38;5;8m | [39m 99  97 114  97 104  99  32 101  109  97 115  32 102 [38;5;8m0076  |[39mcarahc emas f[38;5;8m |[39m▄▄▄▄▄▄▁▄▄▄▄▁▄
[38;5;8m [39m 32 110 101 101 119 116 101  98   32 115 114 101 116 [38;5;8m00a9  |[39m neewteb sret[38;5;8m |[39m▁▄▄▄▄▄▄▄▁▄▄▄▄[38;5;8m | [39m 32 110 101 101 119 116 101  98   32 115 114 101 116 [38;5;8m0083  |[39m neewteb sret[38;5;8m |[39m▁▄▄▄▄▄▄▄▁▄▄▄▄
[38;5;8m [39m 32 115 105  32 115 116 120 101  116  32 101 104 116 [38;5;8m00b6  |[39m si stxet eht[38;5;8m |[39m▁▄▄▁▄▄▄▄▄▁▄▄▄[38;5;8m | [39m 32 115 105  32 115 116 120 101  116  32 101 104 116 [38;5;8m0090  |[39m si stxet eht[38;5;8m |[39m▁▄▄▁▄▄▄▄▄▁▄▄▄
[38;5;8m [39m 32 121 108 116 110 101 105  99  105 102 102 117 115 [38;5;8m00c3  |[39m yltneiciffus[38;5;8m |[39m▁▄▄▄▄▄▄▄▄▄▄▄▄[38;5;8m | [39m 32 121 108 116 110 101 105  99  105 102 102 117 115 [38;5;8m009d  |[39m yltneiciffus[38;5;8m |[39m▁▄▄▄▄▄▄▄▄▄▄▄▄
[38;5;8m [38;5;10m                            [39m 10   46 104 103 105 104 [38;5;8m00d0  |[38;5;10m       [39m..hgih[38;5;8m |[38;5;10m░░░░░░░[39m▁▂▄▄▄▄[38;5;8m | [38;5;10m101 116  32 115 105 104  84 [39m 10   46 104 103 105 104 [38;5;8m00aa  |[38;5;10met sihT[39m..hgih[38;5;8m |[38;5;10m▄▄▁▄▄▄▃[39m▁▂▄▄▄▄
[38;5;8m [38;5;10m                                                     [38;5;8m00d6  |[38;5;10m             [38;5;8m |[38;5;10m░░░░░░░░░░░░░[38;5;8m | [38;5;10m110 105  32 121 108 110 111  32  115 105  32 116 120 [38;5;8m00b7  |[38;5;10mni ylno si tx[38;5;8m |[38;5;10m▄▄▁▄▄▄▄▁▄▄▁▄▄[39m
[38;5;8m [38;5;10m                                                     [38;5;8m00d6  |[38;5;10m             [38;5;8m |[38;5;10m░░░░░░░░░░░░░[38;5;8m | [38;5;10m 32 100 110 111  99 101 115  32  115 105 104 116  32 [38;5;8m00c4  |[38;5;10m dnoces siht [38;5;8m |[38;5;10m▁▄▄▄▄▄▄▁▄▄▄▄▁[39m
[38;5;8m [39m115 105  32 101 114 101  72 [38;5;10m                         [38;5;8m00d6  |[39msi ereH[38;5;10m      [38;5;8m |[39m▄▄▁▄▄▄▃[38;5;10m░░░░░░[38;5;8m | [39m115 105  32 101 114 101  72 [38;5;10m 32   46 101 108 105 102 [38;5;8m00d1  |[39msi ereH[38;5;10m .elif[38;5;8m |[39m▄▄▁▄▄▄▃[38;5;10m▁▂▄▄▄▄[39m
[38;5;8m [39m111  99  32 101 114 111 109  32  101 109 111 115  32 [38;5;8m00dd  |[39moc erom emos [38;5;8m |[39m▄▄▁▄▄▄▄▁▄▄▄▄▁[38;5;8m | [39m111  99  32 101 114 111 109  32  101 109 111 115  32 [38;5;8m00de  |[39moc erom emos [38;5;8m |[39m▄▄▁▄▄▄▄▁▄▄▄▄▁
[38;5;8m [39m 32 111 116  32 116 120 101 116   32 110 111 109 109 [38;5;8m00ea  |[39m ot txet nomm[38;5;8m |[39m▁▄▄▁▄▄▄▄▁▄▄▄▄[38;5;8m | [39m 32 111 116  32 116 120 101 116   32 110 111 109 109 [38;5;8m00eb  |[39m ot txet nomm[38;5;8m |[39m▁▄▄▁▄▄▄▄▁▄▄▄▄
[38;5;8m [38;5;10m        [39m 46 112 117  32 116 105   32 112  97 114 119 [38;5;8m00f7  |[38;5;10m  [39m.pu ti parw[38;5;8m |[38;5;10m░░[39m▂▄▄▁▄▄▁▄▄▄▄[38;5;8m | [38;5;10m        [39m 46 112 117  32 116 105   32 112  97 114 119 [38;5;8m00f8  |[38;5;10m  [39m.pu ti parw[38;5;8m |[38;5;10m░░[39m▂▄▄▁▄▄▁▄▄▄▄
